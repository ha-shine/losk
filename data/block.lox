var name = "Shine";

{
    print "Hello, " + name;
    var name = "John";
    print "Hello, " + name;
}

print "Hello, " + name;

{
    var otherName = "John";
    name = otherName;
}

print "Hello, " + name;