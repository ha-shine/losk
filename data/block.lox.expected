Hello, Shine
Hello, John
Hello, Shine
Hello, John
